modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts

```diff
@@ -162,6 +164,8 @@ export default {
 	'jira-components': 'Jira Components',
 	'key': 'Key',
 	'last-name': 'Last Name',
+	'last-used-date': 'LAST USED DATA',
+	'latest-build': 'LATEST BUILD',
```

Comment:
```suggestion
	'last-used-date': 'Last Used Date',
	'latest-build': 'Latest Build',
```

@Elvinho I don't think it's a good idea to put all translations uppercase, since we can do that directly in the 

`i18n.translate('latest-build').toUpperCase();`

That way we follow the pattern used here